cassiopeia/pom.xml

```diff
@@ -20,7 +20,8 @@
         <tomcat.server.id>TomcatServer</tomcat.server.id>
 
         <admin.password>admin</admin.password>
-
+        <pullrequest></pullrequest>
+        <buildNumber></buildNumber>
```

Comment:
this also needs the buildnumber plugin which I don't believe is included right now.